Packs/EclecticIQ/TestPlaybooks/playbookEclecticIQTest.yml

```diff
@@ -0,0 +1,129 @@
+id: 716af962-8b78-4e09-864d-ded6f471342b
+version: 22
+vcShouldKeepItemLegacyProdMachine: false
+name: test
+starttaskid: "0"
+tasks:
+  "0":
+    id: "0"
+    taskid: 469b1f2a-d699-4cc0-8924-2eef1bcdbcae
+    type: start
+    task:
+      id: 469b1f2a-d699-4cc0-8924-2eef1bcdbcae
+      version: -1
+      name: ""
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "1"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 440,
+          "y": 20
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "1":
+    id: "1"
+    taskid: 984b41c5-91e8-4f09-8372-5bb0c54a4a56
+    type: regular
+    task:
+      id: 984b41c5-91e8-4f09-8372-5bb0c54a4a56
+      version: -1
+      name: lookup_observables
+      description: Lookup observables from EclecticIQ Intelligence Center Platform
+      script: EclecticIQ|||lookup_observables
```

Comment:
```suggestion
      script: |||lookup_observables
```

Reply:
You can choose between two options: 1) EclecticIQ platform|||lookup_observables
2)|||lookup_observables

You need to link the command to the specific integration which means to write the integration's name as appears in the yml file or to omit the name as you can see in the second option.

Reply:
Updated as per the comment